### PR TITLE
adding library

### PR DESCRIPTION
**Pull request by IPENGR**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required):https://github.com/IPENGR/AvatarJS.git
- Official website (optional, not the repository):http://techprep.me
- NPM package url (optional):
- GitHub / Bitbucket popularity (required):
  - Count of stars:0
  - Count of watchers:1
  - Count of forks:0
- NPM download stats (optional):
  - Downloads in the last day:
  - Downloads in the last week:
  - Downloads in the last month:
# Essential checklist
- [x] I'm the author of this library
  - [x] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [ ] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@cdnjs/intern2 please help review this PR as your practice, thanks.

**Comment by kennynaoh** [comment]
Hi @IPENGR,
I think the library information you mention should be more clear, like `Git repository` or `NPM package url`.
Thanks!

File: ajax/libs/AvatarJS/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "AvatarJS",
+  "filename": "avatarServices.js",
+  "version": "0.2.0",
+  "description": "AvatarJS project for cdn",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/IPENGR/AvatarJS.git",
+    "basePath": "dist/",
```
Comment (pvnr0082t) [inline ajax/libs/AvatarJS/package.json]: We can just use `"dist"`.

File: ajax/libs/AvatarJS/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "files": [
+      "avatarServices.js"
```
Comment (pvnr0082t) [inline ajax/libs/AvatarJS/package.json]: Let's use `**/*` here.

File: ajax/libs/AvatarJS/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "author": "Innovapath",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/IPENGR/AvatarJS.git"
+  },
+
+  "dependencies": {
+
+  },
```
Comment (pvnr0082t) [inline ajax/libs/AvatarJS/package.json]: If `dependencies` field is blank, let's remove it.

File: ajax/libs/AvatarJS/package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  "devDependencies": {
```
Comment (pvnr0082t) [inline ajax/libs/AvatarJS/package.json]: We don't need `devDependencies` in package.json. I think we can remove this field.

**Comment by PeterDaveHello** [comment]
Any detail about the lib you are going to add? @IPENGR please fill the items in the comment template.

**Comment by IPENGR** [comment]
Could not found the lib

**Comment by PeterDaveHello** [comment]
@IPENGR we need to know some of the info about a lib to host it.

**Comment by IPENGR** [comment]
this library used for educational site, we require cdnjs for this library

**Comment by PeterDaveHello** [comment]
@IPENGR you still need to respect the conventions here, this is not a personal project for personal usage, for public usage, we need to know more about a lib.

**Comment by IPENGR** [comment]
ok, we are releasing 0.2 version of my project lib,this lib written in angularjs ,

**Comment by IPENGR** [comment]
hi, I unable to found my library ,still need any info about lib?

**Comment by PeterDaveHello** [comment]
@IPENGR You should fill the info in the template.

**Comment by IPENGR** [comment]
hi, I added a README.md template in our project ,to know info about  lib.

**Comment by PeterDaveHello** [comment]
@IPENGR You should provide the info in the first comment.

**Comment by IPENGR** [comment]
hi, I added the info about lib in first comment and added readme also, ThankYou!

**Comment by PeterDaveHello** [comment]
@cdnjs/intern2 please help review it.

**Comment by IPENGR** [comment]
can I know when the lib will approved?

**Comment by PeterDaveHello** [comment]
@IPENGR the popularity of this lib seems to be not enough to be hosted on our CDN, I'll respectfully suggest you to read our document first for the requirement to host a lib, thanks.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by PeterDaveHello** [comment]
Close due to no response.

**Pull request closed without merging**